src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -626,26 +623,68 @@ private void InitializeOutputPane()
             cancellationToken.ThrowIfCancellationRequested();
             OutputToOutputWindow($"Getting project information - done (took {DateTimeOffset.UtcNow - start})");
 
+            ForceLoadProjectsWhoseDesignTimeBuildFailed(projectInfos);
+
+            CreateDeferredProjects(projectInfos, cancellationToken);
+
+            CallWithTimingLog("Pushing to workspace", FinishLoad);
+        }
+
+        private void ForceLoadProjectsWhoseDesignTimeBuildFailed(IReadOnlyDictionary<string, DeferredProjectInformation> projectInfos)
+        {
+            OutputToOutputWindow($"Loading projects whose design time builds failed - start");
+            var start = DateTimeOffset.UtcNow;
+            var solution4 = (IVsSolution4)_vsSolution;
+            var solution7 = (IVsSolution7)_vsSolution;
+            var guidsOfProjectsThatFailed = projectInfos.Where(pi => DesignTimeBuildFailed(pi.Value) && !solution7.IsDeferredProjectLoadAllowed(pi.Key)).Select(pi => GetProjectGuid(pi.Key)).ToArray();
+            OutputToOutputWindow($"\tForcing load of {guidsOfProjectsThatFailed.Length} projects.");
+            OutputListToOutputWindow("\tIncluding ", projectInfos.Where(pi => DesignTimeBuildFailed(pi.Value) && !solution7.IsDeferredProjectLoadAllowed(pi.Key)).Select(pi => pi.Key));
```

Comment:
I deleted your comment!?!  I'll extract the local.  Didn't before because of the `Conditional("DEBUG")`.